ocaml/stdlib/sys.mli

```diff
@@ -415,7 +415,7 @@ val runtime_warnings_enabled: unit -> bool
 
 (** {1 Optimization} *)
 
-external opaque_identity : 'a -> 'a = "%opaque"
+external[@layout_poly] opaque_identity : ('a : any). 'a -> 'a = "%opaque"
```

Comment:
Is it worth adding a test for @Ekdohibs's question? That is, it might be good to test that a use of `opaque_identity` that crosses layouts is rejected.